app/javascript/src/js/formValidations.js

```diff
@@ -1,121 +1,39 @@
 /* eslint-disable */
```

Comment:
We should hopefully be able to re-enable eslint after this clean up 💪

Reply:
I had this error `import/no-extraneous-dependencies` with this line `import { Application } from 'stimulus';` so i have kept eslint disable for this one line only.

following the linter instructions to npm install introduced other errors, as did yarn add. so i removed both of these installs and disabled linting on that line instead.